docs/aws-config.md

```diff
@@ -22,10 +22,6 @@ If you do not have the `aws` CLI locally installed, follow the [user guide][5] t
 
 1. Move the `velero` binary from the Velero directory to somewhere in your PATH.
 
-_We strongly recommend that you use an [official release](https://github.com/heptio/velero/releases) of Velero. The tarballs for each release contain the
```

Comment:
I would probably leave this recommendation in place, though the part about the YAML is no longer relevant.  It's still improtant that `master` is not stable though. So something like (pls wordsmith):

```
We strongly recommend that you use an [official release](https://github.com/heptio/velero/releases) of
Velero. The tarballs for each release contain the `velero` command-line client. The code in the master branch
of the Velero repository is under active development and is not guaranteed to be stable. Use at your own risk!_	
```

Reply:
I like this wording, and can't really think of what I'd change. Maybe dropping `Use at your own risk!`, but at the same time, it's explicit.

Reply:
:+1: wfm